### ISSUE TITLE
Missing notifications

### ISSUE THREAD

**Issue by dutch-erik** [cyberjunky/home-assistant-p2000#23]
It looks like I'm missing some notifications:

# P2000
sensor:
  - platform: p2000
    regios: 17
    disciplines: 1,2,3,4
    radius: 3500
    scan_interval: 60
    latitude: 51.xxx
    longitude: 4.xxx`

Notifications:
![Clipboard01](https://user-images.githubusercontent.com/62548287/84495381-5db8d200-acab-11ea-8d44-f057e113a05a.jpg)

History HA:
![Clipboard02](https://user-images.githubusercontent.com/62548287/84495394-64474980-acab-11ea-9e4f-5e970ab745bf.jpg)

Is there also a way to show the Lifeliner (Traumahelicopter) notifications?

**Comment by cyberjunky** [comment]
Which website is that?
Maybe they are not gathered/displayed on livep2000.nl which we use for integration.
I cannot look back in time to check.

**Comment by dutch-erik** [comment]
I've indeed seen it depends on the website. 
I also checked https://feeds.livep2000.nl/ and noticed there were 2 notifications, but I forgot to make a screenshot and can't look back either.
Otherwise leave it for now and I'll come back when same kind of thing happens again.

**Comment by AlwinHummels** [comment]
I think the site was http://www.alarmfase1.nl/

**Comment by hqmatics** [comment]
I've got the same problem with my capcode. Got 2 sensors configured:

`
  - platform: p2000
    name: "P2000 Friesland"
    regios: 2
    disciplines: 1,2,3,4
    scan_interval: 30
    icon: mdi:fire-truck
  - platform: p2000
    name: "P2000 Damwald"
    regios: 2
    disciplines: 1
    scan_interval: 10
    capcodes: 303192
    icon: mdi:fire-truck
`
Changed the capcode to another capcode and that worked.


Messages for capcode `303192` do not show up in both sensors.
Seems to be because there are no location nodes in the feed for this capcode.

![p2000-sensor](https://user-images.githubusercontent.com/1026950/91975297-79b69780-ed1f-11ea-9d09-d8d71ccdad5e.png)
